src/appleseed-max-impl/appleseedinteractive/interactivesession.h

```diff
@@ -55,6 +55,8 @@ class InteractiveSession
     void reininitialize_render();
     void end_render();
 
+    InteractiveRendererController* get_render_controller();
```

Comment:
Make const, I think that should be possible.

Reply:
It doesn't compile with const - it calls `InteractiveRendererController::schedule_update(std::unique_ptr<ScheduledAction> updater)` on this pointer.